services/order.py

```
@@ -0,0 +1,44 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket
+
+
+@transaction.atomic
+def create_order(
+    tickets: list[dict],
+    username: str,
+    date: datetime = None
+) -> Order:
+    user, _ = get_user_model().objects.get_or_create(
+        username=username
+    )

```

This uses get_or_create to obtain the user. The project checklist and previous review require that you do not create users implicitly here: "Do not use `get_or_create` for the user; fetch with `get_user_model().objects.get(username=username)` as the task expects creating an Order for an existing user." Replace this with a strict lookup (get) so a missing user surfaces as an error instead of being created silently.